prometheus.lua

```diff
@@ -855,18 +866,18 @@ do
 end
 
 -- Public function to register a counter.
-function Prometheus:counter(name, help, label_names)
-  return register(self, name, help, label_names, nil, TYPE_COUNTER)
+function Prometheus:counter(name, help, label_names, lookup_max_size)
```

Comment:
Unless this something that we would expect many users have a need to set separately for each metric, I would suggest to instead add this as a global parameter passed as part `Prometheus.init` (alongside `error_metric_name` and `sync_interval`)

Reply:
Global setting is ok.